File: Bootcamp_Python/userinput.py
# -*- coding: utf-8 -*-
"""
Created on Thu Jul  9 09:39:01 2020

@author: Cory
"""

import enchant
d = enchant.Dict("en_US")



def spellcheck():
    word = input("Type a word ")
    while not d.check(word):
        word=input("Type a better word ")
        print("Congratulations you typed {} correctly".format(word))


def getWord():
    print("What is your name?")
    x=input()
    print("Hello {}".format(x))

File: bootcamp_R_projects/userinput.R
library(hunspell)


spellcheck<-function(){
  word<-readline("Type a word ")
  while (!hunspell_check(word)){
    word<-readline("Check your spelling and try again")
  }
  cat("Congratulations on spelling",word,"correctly")
}


getName<- function(){
  name<-readline(prompt="What is your name?")
  cat("Hello", name, "have a nice day")
}